rrmngmnt/storage.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import shlex
+from rrmngmnt.service import Service
+
+logger = logging.getLogger(__name__)
+
+LV_CHANGE_CMD = 'lvchange -a %s %s/%s'
+
+
+class NFSService(Service):
+    """
+    Storage management class to maintain NFS services
+    """
+
+    def mount(self, source, target=None, opts=None):
+        """
+        Mounts source to target mount point
+
+        __author__ = "ratamir"
+        :param source: Full path to source
+        :type source: str
+        :param target: Path to target directory, if it is omitted then
```

Comment:
if it is omitted then temporary folder is created and it is the responsible of the caller to remove it at the end ->
if omitted, a temporary folder is created instead